modules/gapi/src/api/gkernel.cpp

```diff
@@ -42,12 +42,14 @@ void cv::gapi::GKernelPackage::removeAPI(const std::string &id)
 
 std::size_t cv::gapi::GKernelPackage::size() const
 {
-    return std::accumulate(m_backend_kernels.begin(),
+    std::size_t pkg_size = std::accumulate(m_backend_kernels.begin(),
                            m_backend_kernels.end(),
                            static_cast<std::size_t>(0u),
                            [](std::size_t acc, const M::value_type& v) {
                                return acc + v.second.size();
                            });
+    pkg_size += m_transformations.size();
```

Comment:
I'm not sure if it's an expected behavior, this is kernel package after all)

Reply:
All the names and stuff are discussable